### PR TITLE
Merge in changes from Pivotal

### PR DESCRIPTION
**Pull request by izgeri**

### CODE REVIEW

File: docs-content/using.html.md.erb [hunk 1]
```diff
@@ -119,9 +111,9 @@ The following command extracts just the ID value that you need.
   $ cf env $APP_NAME | grep authn_login | awk '{print $NF}' | sed 's/host\///g; s/"//g'
   ```  
 
-#####<a id="update-policy"></a>6 - Update Conjur Policy Grants for this Application      
+####<a id="update-policy"></a>6 - Update Conjur Policy Grants for this Application
```
Comment (doodlesbykumbi) [inline docs-content/using.html.md.erb]: Step 6 ?

Comment (doodlesbykumbi) [inline docs-content/using.html.md.erb]: Step 6 ?

File: docs-content/index.html.md.erb [hunk 2]
```diff
@@ -58,8 +55,8 @@ The following table provides version and version-support information about Cyber
         <td>vYOUR-COMPATIBLE-OPS-MAN-VERSION (Ex. v1.8.x, v1.9.x, and v1.10.x)</td>
     </tr>
     <tr>
-        <td>Compatible Elastic Runtime version(s)</td>
-        <td>vYOUR-COMPATIBLE-ELASTIC-RUNTIME-VERSION (Ex. v1.8.x, v1.9.x, and v1.10.x)</td>
+        <td>Compatible Pivotal Application Service version(s)</td>
+        <td>vYOUR-COMPATIBLE-Pivotal-Application-Services-VERSION (Ex. v1.8.x, v1.9.x, and v1.10.x)</td>
```
Comment (doodlesbykumbi) [inline docs-content/index.html.md.erb]: PIVOTAL-APPLICATION-SERVICE ?

Comment (izgeri) [reply docs-content/index.html.md.erb]: So this is a placeholder, and my next PR will replace it.

File: docs-content/using.html.md.erb (hunk 1 above)
Comment (izgeri) [reply docs-content/using.html.md.erb]: Yeah, this should be updated. I can't change what Pivotal has, but I can include this in my next PR too.

**Pull request merged**